cirq/testing/circuit_compare.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Tuple, Sequence, TYPE_CHECKING
+from typing import Tuple, Sequence, TYPE_CHECKING, Optional
```

Comment:
nit alphabetical?

Reply:
Acknowledged. I think this is a bit of a lost cause until we add yapf tooling to fix it automatically. It's not consistent anywhere.